### DISCUSSION TITLE
Pink material when using GLTFImport and GLTFAsset with Load() method after build mobile device

### DISCUSSION THREAD

**Discussion by LeNhuHoang** [atteneder/glTFast#713] [Q&A]
Hi guy, I'm having trouble importing .gltf file from Google Cloud Storage using ```GLTFImport.Load()``` method or ```GLTFAsset.Load()```

I had a Google Could Storage containing Draco compressed file. Went I used ```Load()``` method to load the file it showed me Pink material I guess it missing material. 

But when I run my app in Unity Editor everything works smoothly. the issue shows up after building a mobile device.

This is image about the bug play on mobile device:
![k06g7f7r](https://github.com/user-attachments/assets/38fa73d7-5aee-413b-a61e-65c16a52c0f5)

This is image play on Unity Editor:

<img width="431" alt="Screenshot 2024-08-18 at 11 17 51" src="https://github.com/user-attachments/assets/2d068bc9-9b9a-42bf-b6ee-efc3e8466690">

I'm stuck in here for almost one week. I need help pls.

Thanks!!!!

**Comment by atteneder** [comment]
Most likely you're missing a specific shader variant in you build.

Make sure to thoroughly read the [documentation about materials and shader variants](https://docs.unity3d.com/Packages/com.unity.cloud.gltfast@6.7/manual/ProjectSetup.html#materials-and-shader-variants).
